exporter/metric/option.go

```diff
@@ -159,3 +163,12 @@ func WithSumOfSquaredDeviation() func(o *options) {
 		o.enableSumOfSquaredDeviation = true
 	}
 }
+
+// WithCreateServiceTimeSeries configures the exporter to use `CreateServiceTimeSeries` for creating timeseries.
+// If this is used, metric descriptors are not exported.
+func WithCreateServiceTimeSeries() func(o *options) {
+	return func(o *options) {
+		o.createServiceTimeSeries = true
+		o.disableCreateMetricDescriptors = true
```

Comment:
This line isn't needed, since you check o.createServiceTimeSeries before creating descriptors

Reply:
Ok, thats fine.  Lets remove the check for `o.createServiceTimeSeries` on line 172.

Reply:
Yes, the one in `metric.go` - it should be removed. Thanks